### PR TITLE
Use conda for arrow libs & headers

### PR DESCRIPTION
**Pull request by tebeka**
@yonidavidson What do you think?

IMO
- It's simpler
- We use the same libraries Python uses

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,15 +65,11 @@ plasma-server:
 		plasma_store -m 1000000 -s $(PLASMA_DB)
 
 run-wtr:
-		PKG_CONFIG_PATH=/opt/miniconda/lib/pkgconfig make
-		PKG_CONFIG_PATH=/opt/miniconda/lib/pkgconfig \
-			LD_LIBRARY_PATH=/opt/miniconda/lib \
-			go run ./_misc/wtr.go -db /tmp/plasma.db -id $(ID)
+		make
```
Comment (yonidavidson) [inline Makefile]: What is this make command for?

File: Dockerfile [hunk 2]
```diff
@@ -2,45 +2,32 @@ FROM ubuntu:18.04
 
 # Tools
 RUN apt-get update && apt-get install -y \
+    g++ \
     gdb \
     git \
-    vim \ 
+    make \
```
Comment (yonidavidson) [inline Dockerfile]: How come we suddenly need to install make?

**Comment by yonidavidson** [comment]
It's much simpler, 
We assume that the conda packages are updated as fast as Arrow right?
we might have a problem if we want to work on bleeding edge version that have not come out yet and might not be available for conda.

Big advantage is that we know that all our packages are the same and that helps us - when I had the segmentation problems I checked to see that maybe the different packages don't cause that.

File: Dockerfile (hunk 2 above)
Comment (tebeka) [reply Dockerfile]: No sure, I'll try without and check.

File: Makefile (hunk 1 above)
Comment (tebeka) [reply Makefile]: To build before running `wtr`, this diff is not great.

File: Dockerfile (hunk 2 above)
Comment (tebeka) [reply Dockerfile]: For running `wtr`. We can do without but I say leave it for now.

**Pull request merged**